aten/src/ATen/native/mkldnn/Relu.cpp

```diff
@@ -50,6 +50,21 @@ Tensor& mkldnn_relu_(Tensor& input) {
   return input;
 }
 
+Tensor mkldnn_gelu(const Tensor& input) {
+  if (input.scalar_type() == ScalarType::BFloat16) {
+    TORCH_CHECK(mkldnn_bf16_device_check(),
+        "mkldnn_relu: bf16 path needs the cpu support avx512bw, avx512vl and avx512dq");
```

Comment:
is it ok, that the function is named `mkldnn_gelu` and this message says `mkldnn_relu`? typo?